roles/network_plugin/kube-ovn/templates/cni-ovn.yml.j2

```diff
@@ -249,6 +249,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
+            - name: NODE_IPS
+              value: "{{ kube_ovn_central_ips }}"
```

Comment:
I needed to add this to my deployment as well to make it work :+1: 

The original installation script also uses these IPs for the `OVN_DB_IPS` in the `ovs-ovn` DaemonSet so I think it might make sense to add them there as well :)

https://github.com/kubeovn/kube-ovn/blob/master/dist/images/install.sh#L2651-L2652